packages/default/scss/window/_layout.scss

```diff
@@ -47,8 +47,7 @@
 
     // Actions
     .k-window-actions {
-        margin-top: add-two(-$button-padding-y, -$button-border-width);
-        margin-bottom: add-two(-$button-padding-y, -$button-border-width);
+        margin: add-two(-$button-padding-y, -$button-border-width);
```

Comment:
Shouldn't horizontal margins be connected to `$button-padding-x` (instead of `-y`), or am I missing something?

Reply:
Button icon is square and uses $button-padding-y for the both directions.